packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -500,12 +500,16 @@ class Route extends EmberObject implements IRoute {
     export default class ApplicationRoute extends Route {
       @action
       loading(transition, route) {
-        let controller = this.controllerFor('foo');
-        controller.set('currentlyLoading', true);
+        const controller = this.controllerFor('foo');
```

Comment:
@nickeidler I'd actually expect that we'll want this to go back to a `let`, Ember tends to default to a liberal let approach (https://madhatted.com/2016/1/25/let-it-be)